modules/dynamic-operator/api/v1alpha1/kuberlogicservice_webhook.go

```diff
@@ -221,3 +238,16 @@ func validateScheduleFormat(schedule string) error {
 	_, err := cron.ParseStandard(schedule)
 	return err
 }
+
+func validateDomain(domain string) error {
+	klsList := &KuberLogicServiceList{}
+	if err := k8sClient.List(context.TODO(), klsList); err != nil {
+		return err
+	}
+	for _, item := range klsList.Items {
+		if item.Spec.Domain == domain {
+			return errors.New(fmt.Sprintf("Domain %s conflicts with tenant: %s", domain, item.Name))
```

Comment:
Take a look `fmt.Errorf` instead of `errors.New(fmt.Sprintf`

Reply:
Got it, thank you